src/components/AppMethodPrimary.vue

```diff
@@ -36,22 +36,48 @@
       </div>
 
     </div>
-    <div v-else-if="selectedFilter === 'order an array'">
+    <div v-else-if="selectedFilter === options[5]">
       <method-choice :options="ordering"/>
     </div>
-    <div v-else-if="selectedFilter === 'return a string'">
+    <div v-else-if="selectedFilter === options[4]">
       <method-choice :options="string"/>
     </div>
-    <div v-else-if="selectedFilter === 'something else'">
+    <div v-else-if="selectedFilter === options[6]">
       <method-choice :options="other"/>
     </div>
   </div>
 </template>
 
 <script>
+import {store} from '../../store/index'
 import { mapState } from 'vuex'
 import MethodChoice from './methods/MethodChoice.vue'
 
+/*function localizedState({$store}, prop) {
+  const curLang = $store.state.curLanguage
+  console.log('state', curLang);
+  return $store.state[curLang][prop];
+}*/
```

Comment:
can this comment be deleted? or were you using it for debugging

Reply:
Yes, this can be removed. I've kept it to show my progress. 
I've started with this to map the state to the component.  e.g. `adding: state => localizedState(state, 'adding')` in the computed properties. Afterwads I've created the mapping function. I would prefer the `mapState` from Vuex but I wasn't able to dynamically change the namespace. Something like `...mapState('en', ['adding',...])` but changing 'en' to a variable wasn't working - maybe wrapping in a function could help but I have no idea how.